### PR TITLE
add-release-workflow

### PR DESCRIPTION
**Pull request by ebebbington**
**Description**

* Addd egg.jsonand release.yml

**Other Notes**

* Manually release when this pr is merged

### CODE REVIEW

File: egg.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "stable": true,
+  "repository": "https://github.com/drashland/deno-drash-middleware",
+  "files": [
+    "./dexter/**/*",
+    "./deps.ts",
+    "./README.md",
+    "./logo.png",
+    "LICENSE",
+    "./paladin/**/*"
+  ]
```
Comment (Guergeiro) [inline egg.json:14]: Why not simply add `**`. If we add more middleware we don't have to add them manually here.

Comment (ebebbington) [reply egg.json:14]: Good idea lol :p

**Comment by ebebbington** [comment]
Also added `.eggignore` so we can still use `**/*` but leave out unnecessary files

File: egg.json (hunk 1 above)
Comment (ebebbington) [reply egg.json]: https://github.com/drashland/deno-drash-middleware/pull/37/commits/9beab8212a7224076e4a8464095f9ccbcc3091ff

**Review by Guergeiro — APPROVED** [review]
Good!

**Pull request merged**